src/Grid/grid.jl

```diff
@@ -206,29 +201,51 @@ function ExclusiveTopology(cells::Vector{C}) where C <: AbstractCell
     I_vertex = Int[]; J_vertex = Int[]; V_vertex = EntityNeighborhood[]   
     cell_neighbor_table = Vector{EntityNeighborhood{CellIndex}}(undef, length(cells)) 
 
-    for (cellid, cell) in enumerate(cells)    
-        #cell neighborhood
-        cell_neighbors = getindex.((vertex_cell_table,), cell_vertices_table[cellid]) # cell -> vertex -> cell
-        cell_neighbors = unique(reduce(vcat,cell_neighbors)) # non unique list initially 
-        filter!(x->x!=cellid, cell_neighbors) # get rid of self neighborhood
-        cell_neighbor_table[cellid] = EntityNeighborhood(CellIndex.(cell_neighbors)) 
+    for (cellid, cell) in enumerate(cells)
+        cell_neighbors = union([Set{Int}(vertex_cell_table[vertex]) for vertex ∈ vertices(cell) if vertex_cell_table[vertex] != cellid]...)
```

Comment:
can we omit the splatting?

Reply:
`reduce(union, ...)`?

Reply:
Already done this locally, but thanks for the Pointer Kristoffer! :)